01-main/packages/frostwire

```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64"
```

Comment:
This is already the default, so not needed.

Reply:
hmm, looks like the DEFVER=1 is required